src/include/common/data_chunk/sel_vector.h

```diff
@@ -21,7 +22,11 @@ class SelectionVector {
 
     SelectionVector() : SelectionVector{DEFAULT_VECTOR_CAPACITY} {}
 
-    bool isUnfiltered() const { return selectedPositions == (sel_t*)&INCREMENTAL_SELECTED_POS; }
+    bool isUnfiltered() const { return selectedPositions == INCREMENTAL_SELECTED_POS.data(); }
+    bool isContiguous() const {
```

Comment:
Should this function be private for now?

Reply:
I don't know. I guess it depends if there are places where isUnfiltered cannot be replaced with `forEach`, but where we want to take advantage of the additional cases isContiguous covers.